src/cluster.jl

```diff
@@ -214,7 +214,7 @@ end
 mutable struct LocalProcess
     id::Int
     bind_addr::String
-    bind_port::UInt16
+    bind_port::Int
```

Comment:
It's because the `bind_port` field will be printed out in hex by default, which is very annoying when debugging :sweat_smile: (I mentioned it in the commit message for 698b808)

Reply:
Ok seems fine by me